run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
Does this mean that you can not run jobs from the app and jobs from the cli on the same instances?